ACE/ace/Condition_Attributes.h

```diff
@@ -22,6 +22,18 @@
 
 #include "ace/OS_NS_Thread.h"
 
+#if defined (ACE_HAS_CONDATTR_SETCLOCK) && defined (ACE_HAS_CLOCK_GETTIME_MONOTONIC)
+#  if !defined (ACE_HAS_POSIX_MONOTONIC_CONDITIONS)
+#    define ACE_HAS_POSIX_MONOTONIC_CONDITIONS
```

Comment:
Shouldn't this be in config-posix.h? And the define below in a config-win32 file?